projects/L4T/devices/Switch/packages/usb-gadget-scripts/assets/usb-gadget.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+manufacturer="Nintendo"
+product="Switch(Lakka)"
```

Comment:
use a placeholder `@DISTRO@` instead of Lakka maybe and then sed the placeholder?

Reply:
This is still very much WIP, and I have plans to do this once it is finished. for a shell in lakka, it works though, but in the end, I want to add mtp as well, problem is there is a kernel issue that breaks ffs gadget I still need to look into.